entrypoint.sh

```diff
@@ -69,7 +86,13 @@ EOF
 
 if [ ! -z $INPUT_SKIP_UNCHANGED_DIGEST ]; then
     export DIGEST=$(cat digest)
-    export REMOTE=$(reg digest "$IMAGE" | tail -1)
+
+    if [ "$REGISTRY" == "docker.pkg.github.com" ]; then
```

Comment:
Huh, nice catch! Didn't really think about this, and it seems I didn't test it either 😅